src/main/java/com/emc/mongoose/storage/driver/kafka/cache/AdminClientCreateFunctionImpl.java

```diff
@@ -0,0 +1,19 @@
+package com.emc.mongoose.storage.driver.kafka.cache;
+
+import lombok.Value;
+import org.apache.kafka.clients.admin.AdminClient;
+
+import java.util.Properties;
+
+@Value
+public final class AdminClientCreateFunctionImpl
+        implements AdminClientCreateFunction {
+
+    Properties properties;
+
+    @Override
+    public final AdminClient apply(final String name) {
+        properties.setProperty("group.id", name);
```

Comment:
According to documentation AdminClient doesn't have field "group.id". It has "client.id".